contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,212 @@
+# Monitoring Kubernetes Health
+
+**Table of Contents**
+
+- [Monitoring Kubernetes Health](#monitoring-kubernetes-health)
+  - [Monitoring the health of Kubernetes with TestGrid](#monitoring-the-health-of-kubernetes-with-testgrid)
+    - [What dashboards should I monitor?](#what-dashboards-should-i-monitor)
+  - [What do I do when I see a TestGrid alert?](#what-do-i-do-when-i-see-a-testgrid-alert)
+    - [Communicate your findings](#communicate-your-findings)
+    - [Fill out the issue](#fill-out-the-issue)
+    - [Iterate](#iterate)
+
+
+## Monitoring the health of Kubernetes with TestGrid
+
+TestGrid is a highly-configurable, interactive dashboard for viewing your test
+results in a grid, see https://github.com/GoogleCloudPlatform/testgrid.
+
+The Kubernetes community has its own instance of TestGrid, https://testgrid.k8s.io/,
+which we use to monitor and observe the health of the project.
+
+Each SIG has its own set of dashboards and each dashboard is composed of
+different end-to-end (e2e) jobs.
+E2E jobs are in turn made up of test stages (i.e., bootstraping a Kubernetes
+cluster, tearing down a Kubernetes cluster) and e2e tests (i.e., Kubectl client
+Kubectl logs should be able to retrieve and filter logs).
+These views allow all different teams to monitor and understand how their areas
+are doing.
+
+We highly enourage any one to take periodically monitor these dashboards.
+If you see that a job or test has been failing please raise an issue with the
+corresponding SIG in either their mailing list of in slack.
+
+Help maintaining tests, fixing broken tests, improving test success rates, and
+overall test improvements are always highly needed throughout the project.
+
+**Note**: It is important that all SIGs periodically monitor their jobs and
+tests. These are used to figure out when to release Kubernetes.
+Furthermore, if jobs or tests are failing or flaking, then pull requests will
+take a lot longer to be merged.
```

Comment:
Might be an opportune time to mention the difference between periodic / presubmit / postsubmit

Reply:
Also possibly mentioning [metrics](https://github.com/kubernetes/test-infra/tree/master/metrics)